helm/cluster-api-provider-azure/templates/rbac.yaml

```diff
@@ -127,6 +128,7 @@ rules:
       - watch
   - apiGroups:
       - exp.infrastructure.cluster.x-k8s.io
+      - infrastructure.cluster.x-k8s.io
```

Comment:
This is already covered here https://github.com/giantswarm/cluster-api-provider-azure-app/blob/3953a5e9d7a3749b56b8540953c47f08060c0721/helm/cluster-api-provider-azure/templates/rbac.yaml#L156-L157

Reply:
removed